homeassistant/components/smappee/sensor.py

```diff
@@ -31,31 +33,31 @@
         "mdi:power-plug",
         ENERGY_WATT_HOUR,
         "power_today",
-        None,
+        ENERGY_WATT_HOUR,
         False,  # cloud only
     ],
     "power_current_hour": [
```

Comment:
For a future PR, we now have a nice way of expressing such rows with SensorEntityDescription (subclassable if you want to also store your `cloud` property). See for example https://github.com/home-assistant/core/pull/54316